modules/images/webp-uploads/fallback.js

```diff
@@ -0,0 +1,73 @@
+( function( d ) {
+	window.webpUploadsFallbackWebpImages = function( media ) {
```

Comment:
```suggestion
window.wpPerfLab = wpPerfLab || {};
( function( d ) {
	window.wpPerfLab.webpUploadsFallbackWebpImages = function( media ) {
```

I suggest putting anything exposed like this in a namespace. 

If you intent to merge to core, you might want to use the `wp` namespace.

Reply:
Sounds good. Updated.